static/js/faculty_carousel.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Slider from 'react-slick';
+
+const carouselDiv = document.querySelector('#faculty-carousel');
+
+class FacultyTile extends React.Component {
+  render() {
+    let nameStr;
+    if (this.props.title) {
+      nameStr = this.props.name + ", " + this.props.title
+    } else {
+      nameStr = this.props.name
+    }
+    let style = {'backgroundImage': 'url('+this.props.imageUrl+')'};
```

Comment:
If the url had a `)` in it, you might get `url())`. I think we can just assume the URL is encoded properly before we get it in this component.